controller/configurations/amazon_prime.py

```diff
@@ -0,0 +1,91 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import traceback as tb
+import utils.cookie_manager as cm
+
+class AmazonPrimeVideo:
+    ''' Amazon Prime Video configuration '''
+
+    def __init__(self):
+        self.browser = None
+        self.path = "C:\\Program Files (x86)\\Google\\Chrome\\chromedriver.exe"
+        self.username = ''
+        self.password = ''
+        self.save_cookies_on_exit = True
+
+    def launch_amazon_prime(self, fresh_start=False):
+        ''' Launch amazon prime login window '''
+
+        self.browser = webdriver.Chrome(self.path)  # open a browser window
+        self.browser.maximize_window()              # maximize it
+
+        self.browser.get('https://www.primevideo.com/') # load primevideo.com
+
+        if not fresh_start: # if this is not a fresh start
+
+            # then add cookies in the browser
+            cookie_manager = cm.CookieManager(self.browser, "amazon_prime")
+            cookie_manager.add_cookies_to_browser()
+            del cookie_manager
+            # refresh page
+            self.browser.get('https://www.primevideo.com/')
+
+        else:   # otherwise go to the sign in page
+
+            sign_in_button_css = 'a._33ixDQ'
+            sign_in = self.browser.find_element_by_css_selector(sign_in_button_css)
+            sign_in.click()
+
+        return 1
```

Comment:
return 1? Does it indicate something later on in the program control flow?

Reply:
Return 0 will be used when we need to change configurations (like on minimizing and closing sevice)
Return 1 will be used for normal execution
Return -1 in case an Exception occurs in the service. The traceback will be printed on the terminal and -1 is returned to the circuit board.
Anything else shouldn't be returned (if it does then we'll have to fix the code)